src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -483,7 +483,27 @@ private void VerifyReturnTypeMatch()
         /// <summary>
         /// Compute the dependencies. Called after binding. 
         /// </summary>
-        public void ApplyDependencyAnalysis()

```

Comment:
>ApplyDependencyAnalysis [](http://example.com/codeflow?start=20&length=23)

nit - this is breaking. 
Maybe keep old one, add [Obsolete] and point it to new one?